Src/FluentAssertions/Numeric/NullableShortAssertions.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Diagnostics;
+
+namespace FluentAssertions.Numeric
+{
+    /// <summary>
+    /// Contains a number of methods to assert that a nullable <see cref="short"/> is in the expected state.
+    /// </summary>
+    [DebuggerNonUserCode]
+    public class NullableShortAssertions : NullableNumericAssertions<short>
+    {
+        public NullableShortAssertions(short? value)
+            : base(value)
+        {
+        }
+
+        private protected override short? CalculateDifferenceForFailureMessage(short expected)
+        {
+            if (Subject < 10 && expected < 10)
+            {
+                return null;
+            }
+
+            var difference = (short?)(Subject - expected);
```

Comment:
Isn't this cast unnecessary?

Reply:
![image](https://user-images.githubusercontent.com/17405336/160296557-1580ec1d-d48f-4267-a232-e3437f3eca20.png)